tagger.js

```diff
@@ -30,6 +30,13 @@
     })();
     // ------------------------------------------------------------------------------------------
     function tagger(input, options) {
+        if (input.length) {
+            if (input.length === 0) {
+                return;
```

Comment:
It doesn't make sense the length is not 0 from the previous `if` check.
I would use:

```javascript
if (input.length === 0) {
    return;
} else if (input.length === 1) {
    input = Array.from(input).pop();
}
```

Reply:
Yep you're right, I copied this over from my combined modifications and tidied it up incorrectly. I'll fix and update.